proposals/littledan-STANDARDS_OUTREACH/README.md

```diff
@@ -0,0 +1,47 @@
+# JavaScript Outreach Groups
+>  Stage 0
+
+## Champion
+
+Daniel Ehrenberg (@littledan)
+
+## Description
+
+[JS Outreach Groups](https://github.com/littledan/js-outreach-groups)
+is an effort to have regular calls with various segments of the
+JavaScript community, to consult with them on emerging standards.
+We're starting with TC39 topics, but I imagine it could be useful for
+other standards as well.
+
+## Required Resources
+
+Resources the Foundation could provide to help:
+- A neutral home to base this outreach from, which participants can trust. It's awkward to have this as a personal repository.
+- Access to attend standards committee meetings for people who are heavily involved in the efforts ("invited expert" can get awkward).
+
+Work involved in this effort includes:
+- Identifying constituencies who would have relevant feedback in standards bodies which isn't reaching those standards bodies; calling them up, comvincing them to discuss
```

Comment:
> Identifying constituencies who would have relevant feedback in standards bodies which isn't reaching those standards bodies; calling them up, **comvincing** them to discuss

Noticed a little typo for **convincing**.

* constituencies here is limited to only members of the foundation or could we identify some certain individuals from outside the Foundation to also give feedback?